src/index.html

```diff
@@ -3,11 +3,350 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Title</title>
+    <title>Eco cosmetics</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="icon" href="./images/logo.svg">
   </head>
-  <body>
-    <h1>Hello Mate Academy</h1>
+  <body class="page">
+    <header class="header header--gap">
+      <div class="header__menu-toggler"></div>
+      <div class="header__logo">
+        <a href="#">
+          <img
+          class="header__img"
+          src="./images/logo.svg"
+          alt="Logo"
+        >
+        </a>
+        <p class="header__title">witch of mountain winds</p>
+      </div>
+      <nav class="header__nav">
+        <a class="header__nav-link" href="#">Shop</a>
+        <a class="header__nav-link header__nav-link--pad" href="#">Contact</a>
+        <a class="header__nav-link" href="#">About</a>
+      </nav>
+      <a href="#" class="header__link">
+        <img
+        class="header__basket"
+        src="./images/bag.svg"
+        alt="basket"
+      >
+      </a>
+    </header>
+
+    <main class="main">
+      <section class="brand">
+        <div class="brand__name">
+          <h1 class="brand__h1">Brand of <br> eco-cosmetics <br> and magic
+            candles
+          </h1>
+          <p class="brand__text">20% of sales at a wild bird shelter</p>
+          <button class="brand__button" type="button">SHOP</button>
```

Comment:
```suggestion
          <button class="brand__button" type="button">Shop</button>
```
Always better to write in lower case and use `text-transform: uppercase in css